mappings/net/minecraft/server/network/ServerConfigurationNetworkHandler.mapping

```diff
@@ -13,6 +13,6 @@ CLASS net/minecraft/class_8610 net/minecraft/server/network/ServerConfigurationN
 	METHOD method_52407 (Lnet/minecraft/server/MinecraftServer$class_7460;)V
 		ARG 1 properties
 	METHOD method_52409 sendConfigurations ()V
-	METHOD method_52410 queueJoinWorldTask ()V
+	METHOD method_52410 endConfiguration ()V
```

Comment:
I dont think this is correct, this method is only called from the unconfig debug command, and is not used during normal login/play. `cancelConfiguration` may be slightly better?

Reply:
Its not wrong, and it is better than queueJoinWorldTask. Unless you have another idea im happy to go with this.

Reply:
Guess will go with this and see mojang changes in snapshot